src/components/InngestCommHandler.ts

```diff
@@ -412,18 +413,20 @@ export class InngestCommHandler<H extends Handler, TransformedRes> {
    */
   public createHandler(): (...args: Parameters<H>) => Promise<TransformedRes> {
     return async (...args: Parameters<H>) => {
+      const timer = new ServerTiming();
```

Comment:
Should this be in the `createHandler` or created within `runStep`? From what I understand, this is created when `serve()` is called, which is fine in serverless, but may have conflicts in serverful envs like Express. I may be wrong with my assumptions here.

Reply:
It's okay that it's here, as it's inside the function returned by `createHandler()`, therefore this is only called when a request comes in, rather than when the request handler is instantiated.